api_docs/changelog.md

```diff
@@ -20,6 +20,21 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 210**
+
+* [`POST /streams/{stream_id}/delete_topic`](/api/delete-topic),
+  [`POST /mark_all_as_read`](/api/mark-all-as-read): These endpoints
+  now include a `complete` boolean field in the success response to
+  indicate if the response processed all messages (`complete: "true"`)
+  or if only some of the targeted messages were processed (`complete:
+  "false"`) before the request timed out. If the request timed out, then
+  a `code` field will also be returned with an error string. As of this
+  feature level, responses with an HTTP status code of 200 will have
+  `result: "success"` in the response. Prior to this feature level,
+  a response with an HTTP status code of 200 would return `result:
+  "partially_completed"` if not all messages targeted by the request
+  were processed; see feature levels 154 and 153 for more details.
```

Comment:
This feels like it's going into a level of detail that we can leave to the endpoints' own docs pages. In general I think the key information for entries in this changelog is:
* the identifiers for anything that was added, was removed, or had its semantics change (so that the reader can efficiently find relevant changes with Ctrl+F);
* a bit more text to help the reader determine if the change is likely relevant to whatever they're currently interested in (and to provide more targets for Ctrl+F);
* the links to the main docs, where we fully describe the old and new API. (By describing the current API, and then in "Changes" sections describing successive diffs to reach the previous APIs.)

Here's a potential version of this entry:
```suggestion
* [`POST /streams/{stream_id}/delete_topic`](/api/delete-topic),
  [`POST /mark_all_as_read`](/api/mark-all-as-read):
  Added a `complete` field in the response to indicate whether
  all of the targeted messages were processed or only some of them,
  along with a `code` field for an error string.
  These replace the use of `result: "partially_completed"`
  (introduced in feature levels 154 and 153), so that these
  endpoints now send a `result` of either `"success"` or `"error"`,
  like the rest of the Zulip API.
```

Reply:
Yeah that proposal lgtm.

Reply:
Thanks for that revision! I updated the API changelog entry for the above with a few small clarifying revisions (noting that `complete` is a boolean field, noting that these are returned in success responses).